src/panels/ConfigPanel/ConfigPanel.tsx

```diff
@@ -194,7 +222,7 @@ export default function ConfigPanel(): JSX.Element {
     setEnableSave(false);
     setConfirmType(null);
     if (errors?.length) {
-      setFormErrors(errors);
+      setInlineError(InlineErrors.FORM, errors);
       return;
     }
     await getConfig(
```

Comment:
I might be missing something, but I can't see why it fetches the config once it saves successfully as all it does after that is close the panel.

If we remove this call then `getConfig` no longer needs to return a promise, so we can avoid the `void` inside `getConfigCallback`.